ui/src/components/forms/FormField.js

```diff
@@ -0,0 +1,12 @@
+class FormField {
+  constructor(id, fieldName, fieldType, fieldDescription, required, options) {
+    this.id = id;
```

Comment:
no need to pass in id i think. just auto generate using uuid upon creation

https://www.npmjs.com/package/uuid

so something like 

```
import { v4 as uuidv4 } from 'uuid';

class FormField {
  constructor(fieldName, fieldType, fieldDescription, required, options) {
    this.id = uuidv4();
    // blablabla
  }
}
```